test-series/0/Validation_040.ipynb

```diff
@@ -0,0 +1,648 @@
+{
```

Comment:
<p>The numbers here are the ones I was referring to above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera-validation/pull/57/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>